gammapy/analysis/tests/test_analysis.py

```diff
@@ -109,6 +112,25 @@ def test_analysis_1d():
     assert_allclose(dnde[-1].value, 5.382879e-21, rtol=1e-2)
 
 
+@requires_data()
+def test_exclusion_region(tmp_path):
+    config = AnalysisConfig.from_template("1d")
+    analysis = Analysis(config)
+
+    skydir = SkyCoord(83, 22, unit="deg", frame="icrs")
+    exclusion_region = CircleSkyRegion(center=SkyCoord(85, 23, unit="deg", frame="icrs"), radius=1 * u.deg)
```

Comment:
Write `SkyCoord("85d 23d")` to be shorter and try to fit on one line (black will break this line otherwise). `frame="icrs"` is the default, no need to specify it.

Reply:
Done